src/test/java/com/answerdigital/answerking/builder/OrderRequestTestBuilder.java

```diff
@@ -3,14 +3,15 @@
 import com.answerdigital.answerking.request.LineItemRequest;
 import com.answerdigital.answerking.request.OrderRequest;
 
-import java.util.ArrayList;
 import java.util.List;
 
 public class OrderRequestTestBuilder {
     private List<LineItemRequest> lineItemRequests;
 
     public OrderRequestTestBuilder withDefaultValues() {
-        this.lineItemRequests = new ArrayList<>();
+        this.lineItemRequests = List.of(
```

Comment:
I suggest we don't do it this way, list.of() creates an immutable list, meaning we won't have the ability to add any extra values to the list in our unit tests.

Reply:
So I was thinking about this, would we need to change the list if we're using `withDefaultValues()` - would we not then use `withLineItemRequests()` should we need to override the list of values?

Reply:
Discussed via Slack, decided we will proceed with `List.of()` 😄